components/atoms/NotificationsCard/notification-card.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { FaRegSmile, FaUserCircle } from "react-icons/fa";
+
+interface NotificationCard {
+  type: "reaction" | "follow" | "collaboration";
+  message: string;
+}
+const NotificationCard = ({ type, message }: NotificationCard) => {
```

Comment:
nit: For consistency with other props interfaces

```suggestion
const NotificationCard = ({ type, message }: NotificationCardProps) => {
```